app/src/format.py

```diff
@@ -92,7 +65,7 @@ def format_guru_cards(
     return response_with_citations + "<h3>Related Guru cards</h3>" + cards_html
 
 
-def _get_bem_documents_to_show(
+def _get_documents_to_show(
```

Comment:
If this is not called by any remaining code, remove it.

Reply:
Removed